### ISSUE TITLE
I want a source order viewer for rearranged content

### ISSUE THREAD

**Issue by aarongustafson** [WebWeWant/webwewant.fyi#173]
Since a screen reader, for example, presents the page to a user based on source order, a layout that visually shuffles the order can fail a WCAG review under 1.3.2: Meaningful Sequence and potentially 1.3.1: Info and Relationships. Flex, grid, floats, and absolute positioning can all do this (along with some table trickery). Interactive controls can be easily tested by tabbing, and tools like Microsoft’s Accessibility Insights already show tab order, but non-interactive content is harder to test easily, and harder to present to stakeholders in a meaningful way.

I filed [a feature request with Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=1415410) and [another with Accessibility Insights](https://github.com/Microsoft/accessibility-insights-web/issues/394). I also created [a bookmarklet that will show you the source order](http://adrianroselli.com/2019/04/reading-order-bookmarklet.html) (and another bookmarklet to go along with it that animates the sequence in case you want to visualize it hands-free).

<hr>

https://webwewant.fyi/wants/64/
